css/css-scroll-snap/scroll-target-padding-002.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and scroll-snap-padding with snapping off (y-axis)</title>
+<link rel='author' title='Elika J. Etemad' href='http://fantasai.inkedblade.net/contact'>
+<link rel='help' href='https://www.w3.org/TR/css-scroll-snap-1/#scroll-margin'>
+<link rel='match' href='scroll-target-001-ref.html'>
+<style type='text/css'>
+  .container {
+    border: solid blue 4px;
+    height: 4em;
+    overflow: auto;
+
+    /* to make failing more obvious */
+    background: 0 1em / 100% 1em linear-gradient(red, red) repeat-x;
+    /* avoid anti-aliasing issues */
+    font: 20px/1 sans-serif;
+  }
+  .container > div {
+    height: 1em;
+  }
+  .container { scroll-padding: 2em 0 1em; } /* snap area is exact fit for snapport */
```

Comment:
Shouldn't this be `scroll-padding: 1em 0 2em;` here?

Reply:
No, see above.